templates/NovaSeq_BCL2FASTQ_Reverse_Complement_Samplesheet.csv

```diff
@@ -57,7 +57,7 @@ if (index > 0) {
 }
 
 ${SAMPLE.NAME##NoSpecialCharacters}
-return token.replaceAll(\"[^a-zA-Z0-9_]\", \"_\")
+return token.replaceAll(\"[^a-zA-Z0-9-]\", \"-\")
```

Comment:
Yes`-` is a 'non-special / valid' character in sample names. This regex replaces everything that is **not** in `[a-zA-Z0-9-]` with a `-`, the not is done by `^`. If we would remove `-` from the list, it would replace `-` with `-` which does not make a lot of sense, although it would also not harm anything.